felix/fv/wireguard_test.go

```diff
@@ -1068,16 +1088,18 @@ var _ = infrastructure.DatastoreDescribe("WireGuard-Unsupported", []apiconfig.Da
 	)
 
 	type testConf struct {
-		WireguardEnabledV4 bool
-		WireguardEnabledV6 bool
+		WireguardEnabledV4        bool
+		WireguardEnabledV6        bool
+		WireguardThreadingEnabled bool
 	}
 	for _, testConfig := range []testConf{
-		{true, false},
-		{false, true},
-		{true, true},
+		{true, false, false},
+		{false, true, true},
+		{true, true, false},
```

Comment:
Given that we don't test anything new here, we should revert the changes in this test, and just pass `false` to `wireguardTopologyOptions`.